src/fwupdtool.md

```diff
@@ -0,0 +1,43 @@
+% fwupdtool(1) @PACKAGE_VERSION@ | standalone firmware update utility man page
+
+NAME
+====
+
+**fwupdtool** — standalone firmware update utility
+
+SYNOPSIS
+========
+
+| **fwupdtool** [CMD]
+
+DESCRIPTION
+===========
+
+This tool allows an administrator to use fwupd plugins directly without using the daemon process.
```

Comment:
Probably need a guide of who/when to use fuwpdmgr vs fwupdtool

Reply:
Added.